README.md

```diff
@@ -46,6 +46,52 @@ docker run -it hyrise
 
 Inside the container, you can then checkout Hyrise and run `./install_dependencies.sh` to download the required submodules.
 
+
+## Development inside Docker container using CLion
+
+The recommended way of developing with CLion on unsupported OS versions is to use the remote debugging capabilities of 
```

Comment:
Maybe add a reference to the tutorial (which I guess you used?)
https://blog.jetbrains.com/clion/2020/01/using-docker-with-clion/#creating-a-docker-container-for-use-with-clion

Reply:
Absolutely. Thought I put this in, but well. :D